assistant/db/connection.py

```diff
@@ -0,0 +1,16 @@
+import sqlite3
+
+from typing import Optional
+
+
+DB_NAME = "assistant"
+DB_PATH = "./db/{db_name}.sqlite"
```

Comment:
This file now only lives inside the container. Upon rebuild the contents will be lost. Please mount the database file in the `docker-compose` files instead of creating it here. The database file should be external to the container, so it can be backed up etc. separately.

Reply:
Never mind, I misunderstood the user AC: the CSV file should be the one to be external, not the sqlite file.